src/transport_ofi.c

```diff
@@ -534,15 +544,18 @@ static inline int atomic_limitations_check(void)
     /* ----------------------------------------*/
 
     int i, ret = 0;
+    long atomicwarn = 0;
+
+    atomicwarn = shmem_util_getenv_long("OFI_ATOMIC_CHECKS_WARN", 0, 0);
```

Comment:
```shmem_util_getenv_long``` isn't quite right here.  You could either say that this is enabled when the env var is defined, or we could add a ```shmem_util_getenv_bool``` function.